scripts/hello_world_kenhkelly.go

```diff
@@ -0,0 +1,28 @@
+package main
```

Comment:
Please read over the requirements for a script file. Issue #1292

Reply:
I read this and then forgot to add it! It has been added. Thanks.

Reply:
:+1: